comments-by-guideline-and-success-criterion.md

```diff
@@ -0,0 +1,1810 @@
+Comments by Guideline and Success Criterion
+-------------------------------------------
+
+The sections that follow are organized according to the principles, guidelines, and success criteria from WCAG 2.2. The text of each item from WCAG 2.2 is copied as quoted text. Following that, the WCAG2ICT guidance is provided. Finally, the “Intent” from Understanding WCAG 2.2 is copied as quoted text; the Task Force makes no substitutions or edits in this text. In visual presentations, the WCAG2ICT guidance is set out in a box with a blue bar to the left, to highlight that this is the content specific to this document.
+
+### Principle 1: Perceivable
+
+From [Principle 1](http://www.w3.org/TR/WCAG22/#perceivable):
+
+> Information and user interface components must be presentable to users in ways they can perceive.
+
+#### Additional Guidance When Applying Principle 1 to Non-Web Documents and Software:
+
+In WCAG 2.2, the Principles are provided for framing and understanding the success criteria under them but are not required for conformance to WCAG. Principle 1 applies directly as written.
+
+#### Guideline 1.1: Text Alternatives
+
+From [Guideline 1.1](http://www.w3.org/TR/WCAG22/#text-alternatives):
+
+> Provide text alternatives for any non-text content so that it can be changed into other forms people need, such as large print, braille, speech, symbols or simpler language.
+
+##### Additional Guidance When Applying Guideline 1.1 to Non-Web Documents and Software:
+
+In WCAG 2.2, the Guidelines are provided for framing and understanding the success criteria under them but are not required for conformance to WCAG. Guideline 1.1 applies directly as written.
+
+Intent from [Understanding Guideline 1.1 in Understanding WCAG 2.2](http://www.w3.org/WAI/WCAG22/Understanding/text-alternatives#intent):
+
+> The purpose of this guideline is to ensure that all non-text content is also available in [text](http://www.w3.org/TR/WCAG22/#dfn-text). "Text" refers to electronic text, not an image of text. Electronic text has the unique advantage that it is presentation neutral. That is, it can be rendered visually, auditorily, tactilely, or by any combination. As a result, information rendered in electronic text can be presented in whatever form best meets the needs of the user. It can also be easily enlarged, spoken aloud so that it is easier for people with reading disabilities to understand, or rendered in whatever tactile form best meets the needs of a user.
+> 
+> _Note:_ While changing the content into symbols includes changing it into graphic symbols for people with developmental disorders and speech comprehension difficulties, it is not limited to this use of symbols.
+
+### Success Criterion 1.1.1: Non-text Content (Level A)
+
+From [Success Criterion 1.1.1](http://www.w3.org/TR/WCAG22/#non-text-content):
+
```

Comment:
Not sure how we do includes from the WCAG repository, as it isn't copying in the text of the SC here.

Reply:
This will be handled in script separate from the port, which we still have to set up.